plugins/zfs/zpool_iostat

```diff
@@ -1,4 +1,7 @@
-#!/bin/bash
+#!/bin/sh
+
+#%# family=auto
+#%# capabilities=autoconf
```

Comment:
I'll rewrite the required autoconf function, unless someone else beats me to it. :)